backend/src/test/java/com/google/blackswan/mock/SimpleAnomalyGeneratorTest.java

```diff
@@ -0,0 +1,120 @@
+package com.google.blackswan.servlets;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static java.util.stream.Collectors.toList;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+
```

Comment:
nit: probably don't need the newline between the import statements here.

Reply:
yep, i got rid of it. thanks!

i didnt get rid of the new line in line 18 as i think it separates the imports that are used for testing versus other imports for data structures and classes that are being tested.